src/shared/components/phase-banner/phase-banner.component.html

```diff
@@ -1,11 +1,32 @@
-<div class="hmcts-header__container">
-  <div class="govuk-phase-banner">
-      <p class="govuk-phase-banner__content"><strong class="govuk-tag govuk-phase-banner__content__tag ">
-              {{type}}
-          </strong>
-          <span class="govuk-phase-banner__text">
-              This is a new service – your <a class="govuk-link" href="https://www.smartsurvey.co.uk/s/CCDSurvey/" target="_blank" rel="noopener noreferrer" aria-label="provide your feedback">feedback</a> will help us to improve it.
-          </span>
+<div class="govuk-width-container app-width-container">
+    <div class="govuk-phase-banner">
+      <ng-container *ngIf="welshLanguageEnabled$ | async">
+        <a href="javascript:void(0)" (click)="toggleLanguage(currentLang == 'cy' ? 'en' : currentLang == 'en' ? 'cy' : 'en')" class="language">{{currentLang == 'cy' ? 'English' : currentLang == 'en' ? 'Cymraeg' : 'English' }}</a>
```

Comment:
These if statements look funky, is this an Angular thing, or could this be simplified. Without task knowledge I'd like to think it could be adjusted to:

currentLang == 'en' ? 'Cymraeg' : 'English'

This would set the test to English for all scenarios other than the currentLang being populated with 'en'.